src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/net/NodeStateManager.java

```diff
@@ -67,12 +67,18 @@ public void updateReceiveTime(final String host, final String graphNode, final l
    */
   public long getLastReceivedTimestamp(String graphNode, String host) {
     final String compositeKey = graphNode + SEPARATOR + host;
-    if (lastReceivedTimestampMap.containsKey(compositeKey)) {
-      return lastReceivedTimestampMap.get(compositeKey);
-    }
+    // Return the last received value or a value that is in the distant past.
+    return lastReceivedTimestampMap.getOrDefault(compositeKey, 0L);
+  }
 
-    // Return a value that is in the distant past.
-    return 0;
+  public SubscriptionStatus getSubscriptionStatus(String graphNode, String host) {
```

Comment:
If this is for test only, can we annotate it as such ?

Reply:
Done